src/Util.js

```diff
@@ -291,7 +284,7 @@ class Util {
         instance.interceptors.response.use(
             (res) => res,
             (err) => {
-                console.log(err.response);
+                console.log(err.response); // eslint-disable-line
```

Comment:
We should add a caption for this value like 'Response error: ...'

Reply:
Will have to refactor this, the logged object is too big anyway